src/Bridge/Symfony/PropertyInfo/Metadata/Property/PropertyInfoPropertyMetadataFactory.php

```diff
@@ -68,6 +68,23 @@ public function create(string $resourceClass, string $name, array $options = [])
             $propertyMetadata = $propertyMetadata->withWritable($writable);
         }
 
+        if (method_exists($this->propertyInfo, 'isInitializable')) {
+            if (null === $propertyMetadata->isInitializable() && null !== $initializable = $this->propertyInfo->isInitializable($resourceClass, $name, $options)) {
+                $propertyMetadata = $propertyMetadata->withInitializable($initializable);
+            }
+        } else {
+            // BC layer for Symfony < 4.2
+            // To be removed in EOF of Symfony 3.4
```

Comment:
I don't understand this comment. If it's a BC layer for Symfony < 4.2, shouldn't it only be removed when we drop support for Symfony < 4.2?

Reply:
Yes we can just drop the last line

Reply:
Yes my bad!